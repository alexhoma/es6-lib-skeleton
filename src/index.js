import {h, render} from "preact"

export const App = (name) => <div>Hello {name}!</div>;