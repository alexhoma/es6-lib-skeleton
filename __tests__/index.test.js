import {h} from "preact";
import {App} from "../src/index"
import {shallow} from "preact-render-spy";


describe("<App>", () => {
   it('should render', () => {
       const tree = shallow(<App name={"world"} />);

       expect(tree).toMatchSnapshot();
   })
});