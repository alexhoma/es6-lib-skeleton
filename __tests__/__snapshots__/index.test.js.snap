// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> should render 1`] = `
preact-render-spy (1 nodes)
-------
<div>
  Hello 
  <undefined></undefined>
  !
</div>

`;
